libraries/AP_Motors/AP_MotorsHeli_Swash.cpp

```diff
@@ -37,12 +37,13 @@ const AP_Param::GroupInfo AP_MotorsHeli_Swash::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("COL_DIR", 2, AP_MotorsHeli_Swash, _swash_coll_dir, COLLECTIVE_DIRECTION_NORMAL),
 
-    // @Param: LIN_SVO
+    // @Param: LIN_SV_ANG
     // @DisplayName: Linearize Swashplate Servo Mechanical Throw
-    // @Description: This linearizes the swashplate servo's mechanical output to account for nonlinear output due to arm rotation.  This requires a specific setup procedure to work properly.  The servo arm must be centered on the mechanical throw at the servo trim position and the servo trim position kept as close to 1500 as possible. Leveling the swashplate can only be done through the pitch links.  See the ardupilot wiki for more details on setup.
-    // @Values: 0:Disabled,1:Enabled
+    // @Description: Set 0 to disable. Set max angle range of servo horn from level. This linearizes the swashplate servo's mechanical output to account for nonlinear output due to arm rotation. This requires a specific setup procedure to work properly. The servo arm must be centered on the mechanical throw at the servo trim position and the servo trim position kept as close to 1500 as possible. Leveling the swashplate can only be done through the pitch links. See the ardupilot wiki for more details on setup.
+    // @Range: 0 60
+    // @Units: deg
     // @User: Standard
-    AP_GROUPINFO("LIN_SVO", 3, AP_MotorsHeli_Swash, _linear_swash_servo, 0),
+    AP_GROUPINFO("LIN_SV_ANG", 3, AP_MotorsHeli_Swash, _linear_swash_servo_ang_deg, 0),
```

Comment:
Strictly using the same index is OK for a change of param type. But I think its better to play it safe and move to a new index.

Reply:
Also there is the whole none versioned param docs thing, so for now we need to keep the old param description around.